src/inverseRobot.js

```diff
@@ -24,7 +24,17 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const trueRobot = {};
+
+  for (const key in robot) {
+    if (trueRobot.hasOwnProperty(robot[key]) === false) {
```

Comment:
hasOwnProperty returns boolean by default, no need to use `=== false` here

```suggestion
    if (trueRobot.hasOwnProperty(robot[key])) {
```